Header header

Appearance appearance
Activities activities
Expressions expressions

FacePoints facePointsInInfraredSpace
FacePoints facePointsInColorSpace

BoundingBox2D faceBoundingBoxInInfraredSpace
BoundingBox2D FaceBoundingBoxInColorSpace

Quaternion faceRotationQuaternion

int64 trackingId
int32 faceFrameFeatures
